tutorials/applications/Using QAOA to solve a MIS problem.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Nice ! Do you get Warnings if the given time durations are not multiples of 4ns?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Only if I build a sequence outside of the loop, which is why I don't do it